recipes-core/images/intel-aero-image.bb

```diff
@@ -56,6 +56,7 @@ GRUB_TIMEOUT = "3"
 
 # librealsense
 IMAGE_INSTALL += "librealsense"
+IMAGE_INSTALL += "librealsense-graphical-examples"
```

Comment:
don't we need to remove the link below to /lib64 after this change? Maybe we need to remove it nonetheless and instruct people to use libdir=/usr/lib64 when building for 64 bits.